spec.bs

```diff
@@ -1224,9 +1223,23 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
     1. [=map/For each=] joiningOrigin → |groups| of |perSignalsUrlGenerator|:
       1. [=list/For each=] |ig| of |groups|:
         1. If |ig|'s [=interest group/bidding url=] is null, [=iteration/continue=].
+        1. Let |directFromSellerSignals| be null.
+        1. Let |buyerDirectFromSellerSignals| be {{undefined}}.
+        1. If |auctionConfig|'s [=auction config/expects direct from seller signals header ad slot=]:
+          1. Let |directFromSellerSignalsKey| be a new [=direct from seller signals key=] with its
+            [=direct from seller signals key/owner=] set to |ig|'s [=interest group/owner=], and

```

Comment:
Hmm, this should actually be the seller origin. Maybe I should rename [=direct from seller signals key/owner=] to something else, like seller?

Reply:
yeah, I guessed it being seller, but confused by the name.
Then my current change which assumes it's owner is wrong. Currently generateBid and scoreAd of the same owner used the same directFromSellerSignals (with only sellerSignals or perBuyerSignals of course).

Reply:
Yes, the signals come from the seller, so the seller is always used for [=direct from seller signals key/owner=]. In component auctions, it should be the component's seller (or top-level for the top-level auction).

Reply:
I see. It actually may make my change slightly easier.
And changing the name from owner to something else like seller would be good. "owner" is confusing.